### ISSUE TITLE
Not getting parser options on build

### ISSUE THREAD

**Issue by morph626** [revolunet/sublimetext-markdown-preview#422]
On a Windows 10, ST3 build 3143 here, and I cannot get the preview to do code highlighting. I get no options to select parser on build. I then tried to set enable_parsers to github, to no effect. Same with setting github_mode to gfm. Any help?

**Comment by facelessuser** [comment]
You will need to be specify step by step what you did, and then clarify your question.  Right now I have no idea how to help because I don't really understand what you are trying to do, and what your problem with it is.

**Comment by morph626** [comment]
Hi, thanks for the quick reply. Reading my description I see why it is confusing. My problem was that I didn't get the option to parse using github on build, so my code blocks didn't get highlighted, and setting the parser to github in settings made no difference.

I found the error, though. When I set my key binding for the markdown_preview command a couple of years ago I apparently set the parser to markdown as an argument, so obviously it would never use github as the parser. I'm and idiot x) Thanks for a really nice plugin.
